src/SDKs/Automation/Management.Automation/Properties/AssemblyInfo.cs

```diff
@@ -4,16 +4,3 @@
 using System.Reflection;
 using System.Resources;
 
-[assembly: AssemblyTitle("Microsoft Azure Automation Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure Automation management operations including the ability to create, update and delete runbooks and schedules.")]
-
-[assembly: AssemblyVersion("3.0.0.0")]
```

Comment:
Please include all assembly attributes in the project.

Reply:
@shahabhijeet I'll send a new PR with the fixes you requested. @finiteattractor Could you please send your test code and I'll add that to the PR and then we can abandon this one.